helm-charts/support/values.yaml

```diff
@@ -100,6 +100,26 @@ prometheus:
       - key: k8s.dask.org/dedicated
         value: worker
         effect: NoSchedule
+    # resources for the node-exporter was set after inspecting cpu and memory
+    # use in a few clusters using prometheus queries from
+    # https://gist.github.com/payam-int/edf977c6af603fee0ce1b05da7792fe7.
+    #
+    # - Memory query:
+    #   100 * (1 - ((avg_over_time(node_memory_MemFree_bytes[10m]) + avg_over_time(node_memory_Cached_bytes[10m]) + avg_over_time(node_memory_Buffers_bytes[10m])) / avg_over_time(node_memory_MemTotal_bytes[10m])))
```

Comment:
Should we have things like  this graphed in grafana?

Reply:
Yes maybe, but also maybe not develop the dashboards for this ourselves. I figure for now, its helpful if there is a quick and easy trace for how to come up with data to decide on other resource requests/limits set in the support chart.